### ISSUE TITLE
Symbol's value as variable is void: url-max-redirections

### ISSUE THREAD

**Issue by the-kenny** [ruediger/emacs-codepad#1]
When I try to use codepad-paste-_, it does the http-request (the past shows up on codepad), but I get the following error-messages in my *Messages_-Buffer:

```
error in process filter: url-http-chunked-encoding-after-change-function: Symbol's value as variable is void: url-max-redirections
error in process filter: Symbol's value as variable is void: url-max-redirections
```

M-x emacs-version says: 
    GNU Emacs 23.0.94.1 (i386-apple-darwin9.8.0, NS apple-appkit-949.54) of 2009-10-26 on braeburn.aquamacs.org - Aquamacs Distribution 2.0preview3

**Comment by the-kenny** [comment]
Hm.. It should be codepad-paste-STAR. Markdown is evil!

**Comment by the-kenny** [comment]
It works if I (setq) url-max-redirections to 0 in my .emacs.

**Comment by neomantic-zz** [comment]
I'm using GNU Emacs 23.1.1 (i386-apple-darwin9.7.0, NS apple-appkit-949.46) of 2009-07-30.

I can confirm that this bug still exists.

**Comment by ruediger** [comment]
Fix in 0e5bb3397e8ec5a4c4edfb77fba02b9a4a673e71
